### PR TITLE
Added required strings for weather command

### PR DESCRIPTION
**Pull request by SuhanKoh**
https://github.com/Frederikam/FredBoat/pull/290#issuecomment-330842648

### CODE REVIEW

File: en_US.properties [hunk 1]
```diff
@@ -130,6 +130,12 @@ brainfuckCycleLimit=Program exceeded the maximum cycle count of {0}
 brainfuckDataPointerOutOfBounds=Data pointer out of bounds\: {0}
 brainfuckInputOOB=Input out of bounds at position\: {0}
 brainfuckNoOutput=\ There was no output
+weatherLoading=Fetching weather ....
```
Comment (napstr) [inline en_US.properties]: Three dots and no space is the general ""convention"" for FredBoat (see CompileCommand for example).

File: en_US.properties [hunk 2]
```diff
@@ -130,6 +130,12 @@ brainfuckCycleLimit=Program exceeded the maximum cycle count of {0}
...
+weatherUsageError=Error occurred with `{0}weather` command, please enter `{0}weather` [location].
```
Comment (napstr) [inline en_US.properties]: Please do not hardcode command triggers/aliases into translation files, the translators would just translate them, and to change them we would have to reset translations.

Please do not use any formatting / markdown / parenthesis or any such kind in the translation files if it's not emphasizing a translatable word or expression. The inserted variables can be formatted perfectly fine in the code reducing the risk of translators fucking up the formatting, and allowing us to change the appearance without resetting translations.

File: en_US.properties [hunk 3]
```diff
@@ -130,6 +130,12 @@ brainfuckCycleLimit=Program exceeded the maximum cycle count of {0}
...
+weatherError=Error retrieving weather for `{0}` \:(
```
Comment (napstr) [inline en_US.properties]: Please do not use any formatting / markdown / parenthesis or any such kind in the translation files if it's not emphasizing a translatable word or expression. The inserted variables can be formatted perfectly fine in the code reducing the risk of translators fucking up the formatting, and allowing us to change the appearance without resetting translations.

File: en_US.properties [hunk 4]
```diff
@@ -130,6 +130,12 @@ brainfuckCycleLimit=Program exceeded the maximum cycle count of {0}
...
+weatherLocation=Here''s what I got for\: `{0}`
```
Comment (napstr) [inline en_US.properties]: Please do not use any formatting / markdown / parenthesis or any such kind in the translation files if it's not emphasizing a translatable word or expression. The inserted variables can be formatted perfectly fine in the code reducing the risk of translators fucking up the formatting, and allowing us to change the appearance without resetting translations.

File: en_US.properties [hunk 5]
```diff
@@ -130,6 +130,12 @@ brainfuckCycleLimit=Program exceeded the maximum cycle count of {0}
...
+weatherDescription=Current weather is `{0}`
```
Comment (napstr) [inline en_US.properties]: Please do not use any formatting / markdown / parenthesis or any such kind in the translation files if it's not emphasizing a translatable word or expression. The inserted variables can be formatted perfectly fine in the code reducing the risk of translators fucking up the formatting, and allowing us to change the appearance without resetting translations.

File: en_US.properties [hunk 6]
```diff
@@ -130,6 +130,12 @@ brainfuckCycleLimit=Program exceeded the maximum cycle count of {0}
...
+weatherTemperature=Temperature is: `{0}`
```
Comment (napstr) [inline en_US.properties]: Please do not use any formatting / markdown / parenthesis or any such kind in the translation files if it's not emphasizing a translatable word or expression. The inserted variables can be formatted perfectly fine in the code reducing the risk of translators fucking up the formatting, and allowing us to change the appearance without resetting translations.

File: en_US.properties [hunk 7]
```diff
@@ -130,6 +130,9 @@ brainfuckCycleLimit=Program exceeded the maximum cycle count of {0}
 brainfuckDataPointerOutOfBounds=Data pointer out of bounds\: {0}
 brainfuckInputOOB=Input out of bounds at position\: {0}
 brainfuckNoOutput=\ There was no output
+weatherLoading=Fetching weather...
+weatherError=Error retrieving weather for {0}
+weatherLocation={0} - {1}
```
Comment (napstr) [inline en_US.properties]: This doesn't need to be translated.

File: en_US.properties [hunk 8]
```diff
@@ -267,3 +269,4 @@ If you want to help us increase the limit or you want to use our non-overcrowded
 {1}\
 \
 Sorry for the inconvenience! You might want to try again later. This message usually only appears at peak time.
+tryLater=Please try again later.
```
Comment (napstr) [inline en_US.properties]: Please add a new line at the end of the file.

**Comment by napstr** [comment]
Oh and this needs a merge/rebase.

File: en_US.properties (hunk 8 above)
Comment (SuhanKoh) [reply en_US.properties]: Ok

**Comment by SuhanKoh** [comment]
Rebased.

**Pull request merged**